feat(content): Conspiracy theorist spaceport news (#9937)
endless-sky/endless-sky: data/human/news.txt

news "conspiracy theorist"
	location
		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
	name
		word

> 			"Conspiracy Theorist"
> 			"Free Thinker"
> 			"Truth Seeker"

Reviewer: News names only capitalise the first letter of the first word.
```suggestion
			"Conspiracy theorist"
			"Free thinker"
			"Truth seeker"
```

	message
		word
			`"`
		word
			"The Rigellian Plague was a fake. The government used it to implant us with tracking microchips."

> 			"If the Rigellian Plague really happened, how come we have so little quarantine facilities now?"

Reviewer 2: ```suggestion
			"If the Rigellian Plague really happened, how come we have so few spaceport quarantine facilities now?"
```
Maybe they don't know the difference. It'd be funnier.

> 			"The Quarg built the pyramids of Earth. There is simply no way that humans could have constructed something so massive back then."

Author: ~~But then I'd have to lock it behind FW, you see~~

> 			"The Quarg built the pyramids of Earth. There is simply no way that humans could have constructed something so massive back then."

Reviewer 3: If anything I'd expect an actual alien invasion to bring way more conspiracy theories, yes :james:

> 			"The Quarg built the pyramids of Earth. There is simply no way that humans could have constructed something so massive back then."

Author: There are news for that in `pug news.txt`.

> 			"The Quarg built the pyramids of Earth. There is simply no way that humans could have constructed something so massive back then."

Reviewer 3: honestly conspiracy theorists knowing about the Pugs would be cool?
the deep know about them, aint hard to get those stories, and the name surely can be found if you dig a bit

			"Spaceship windows are really just screens designed to show planets as round. One day, I will build my own ship to prove they are flat."
			"The Quarg are not aliens, just actors in body suits. There isn't enough sunlight near gas giants for a species to evolve."
			"Hyperdrives were given to the government by the Quarg. If it was our technology, we would have improved it by now."
			"Aliens took our people to the Deep before hyperdrives were invented to experiment on them. This is how they created the Alphas."

> 			"Republic Intelligence has bird-like surveillance drones in every city. There is one in front of my window every morning."

Reviewer 4: ```suggestion
			"Republic Intelligence has bird-like surveillance drones in every city. There's one in front of my window every morning."
```

> 			"The Quarg are running the government. There is no way such a powerful species wouldn't be able to control us."

Reviewer 4: ```suggestion
			"The Quarg are running the government. There's no way such a powerful species wouldn't be able to control us."
```

			"A friend of mine caught a horrible disease after visiting a Quarg ring. She had rashes and carbuncles all over her face."

> 			"If spaceships have drag, there must be air in the 'vacuum of space'..."

Reviewer 2: ~~Unrealistic.~~

		word
			"" 2
			" Wake up, sheeple!"
			" Do your own research!"
			" If you still don't see the truth, I feel sorry for you."
			" Leave the Matrix!"

> 			" You are being lied to!"

Reviewer 5: ```suggestion
			" You are being lied to!"
			" Break the conditioning!"
```
